pyanaconda/modules/users/installation.py

```diff
@@ -195,7 +195,7 @@ def run(self):
                 f.write(
                     '# This file has been generated by the Anaconda Installer.\n'
                     '# Allow root to log in using ssh. Remove this file to opt-out.\n'
-                    'PERMITROOTLOGIN="-oPermitRootLogin=yes"\n'
+                    'PermitRootLogin yes"\n'
```

Comment:
I think you have a stray double quote " there?

Reply:
Right. Will fix in next push.